packages/enterprise/src/license.ts

```diff
@@ -739,6 +737,9 @@ export async function licenseInit(
             // Use the cache
             license = mongoCache.toJSON();
             license.usingMongoCache = true;
+            verifyLicenseInterface(license);
+            keyToLicenseData[key] = license;
+            keyToCacheDate[key] = new Date();
```

Comment:
This logic is now only running within the else block.  Before it was running after both if and else.  Is that intentional?

Reply:
It is done within `updateLicenseFromServer` when it calls `verifyAndSetServerLicenseData` so hence it is done in both branches.